app/javascript/elm/src/Main.elm

```diff
@@ -295,8 +295,12 @@ update msg model =
             in
             ( subModel2, Cmd.batch [ subCmd1, subCmd2 ] )
 
-        ToggleCrowdMap ->
-            ( { model | isCrowdMapOn = not model.isCrowdMapOn }, Ports.toggleCrowdMap (not model.isCrowdMapOn) )
+        ToggleCrowdMap newValue ->
+            if model.isCrowdMapOn == newValue then
+                ( model, Cmd.none )
```

Comment:
What happens if you delete this branch of if else and always execute line 303 - does the behaviour changes? Same for the other toggles. Maybe it's not worth having it. I'm not sure, we can chat about it.

Reply:
I wanted to cause no additional unnecessary action. yep, let's talk